docs/6.x/pack.md

```diff
@@ -131,6 +131,59 @@ docker run -e OPS_URL=<opscenter url> \
     by setting `--state-dir`. You can use unique temporary directory
     to avoid sharing state between builds, or use parallel builds instead.
 
+## Vendoring
```

Comment:
Are you planning to backport to 6.x?

Reply:
No, forgot to add a note that this new resource is only supported in 7.0.

Reply:
If it's only in 7.x I would fork the docs and only put these updates there.